code/index.html

```diff
@@ -85,7 +86,7 @@ <h3 class="site-big-news-headline">
         </div>
       </section>
 
-      <a id="more_fake_news"></a>
+      <a id="more_fake_news"><h2>More Fake News</h2></a>
```

Comment:
Very nice indentations which makes is easy to read! Good naming of classes and ID. It feels like there could be one less div on the cards just to make the HTML easier and shorter but I don't know the code and functions like you. Just a thought. :)